tests/System.Text.Primitives.Tests/MemoryProtection/NativeMemory.Default.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text.Primitives.Tests.MemoryProtection
+{
+    public static partial class NativeMemory
+    {
+        // for non-Windows systems
+        private static INativeMemory AllocateWithoutDataPopulationDefault(int cb)
```

Comment:
What is cb?

Reply:
"cb" is shorthand for "count of bytes", it's Win32 speak for "size".

Reply:
`byteCount`? Can't use a Win32ism with the comment "for non-Windows systems" above it - that's just trollin' :)

Reply:
Yup, similarly, `cch` = "count of chars". Though this parameter name wasn't chosen because I'm in love with Hungarian notation. :) It was simply to match the already existing parameter name on `Marshal.AllocHGlobal`.